sdk/tables/azure-data-tables/tests/test_table_entity_cosmos.py

```diff
@@ -742,6 +750,7 @@ def test_get_entity_with_special_doubles(self, resource_group, location, cosmos_
             self._tear_down()
             self.sleep(SLEEP_DELAY)
 
+    @pytest.mark.skip("Merge operation fails from Tables SDK")
```

Comment:
Please update the string to contain the link https://github.com/Azure/azure-sdk-for-python/issues/13844